recipe/meta.yaml

```diff
@@ -52,7 +52,7 @@ requirements:
     - libxml2 2.9.*
     - openjpeg 2.3.*
     - pango 1.40.*
-    - perl 5.22.2.1
```

Comment:
It is costly for the CIs to support multiple builds like that.
At the moment `conda-forge` does not have a clear PERL police, I would be OK to pin to a modern PERL, like 5.20.3.1, but we cannot afford to build all 3 of them.

Reply:
I understand the overhead of supporting multiple builds. I included all of them because conda-smithy suggested it during rerendering. What I actually need is a build for 5.22.0 as imagemagickis a dependency for nullarbor and bioconda currently pin perl to that version. 

Would it be okey if we only also add a build for 5.22.0, i.e. take away 5.20.3.1?

Reply:
`5.22.0` is a downgrade from  `5.22.2.1` and, based on what you said above, you don't need `5.20.3.1`.

So let's re-pin to `5.22.0` for build number 3 but later we would need to pin again for the latest PERL available to be consistent with the rest of the conda-forge stack.